objectherkenning_openbare_ruimte/performance_evaluation_pipeline/metrics/yolo_to_coco.py

```diff
@@ -0,0 +1,139 @@
+# Adapted from: https://www.kaggle.com/code/siddharthkumarsah/convert-yolo-annotations-to-coco-pascal-voc?scriptVersionId=123233495&cellId=1
+
+import json
+import os
+import pathlib
+from typing import Dict, List, Tuple, Union
+
+from PIL import Image
+
+# Define the categories for the COCO dataset
+CATEGORIES = [
```

Comment:
Is it possible to use the enum you created?

Reply:
Ah yes :)